cdisc_rules_engine/dataset_builders/variables_metadata_with_define_dataset_builder.py

```diff
@@ -28,16 +29,26 @@ def build(self):
         define_variable_has_codelist,
         define_variable_codelist_coded_values
         """
+        vx = kwargs.get("validate_xml", "Y")
+        jm = kwargs.get("join_method", "left")
```

Comment:
I think we agreed that we don't want join_method to be an option here?

Reply:
Removed join_method.